src/adagrams.js

```diff
@@ -1,15 +1,140 @@
+const LETTER_POOL = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+
+function getRandomProperty() {
+  const keys = Object.keys(LETTER_POOL);
+
+  return keys[Math.floor(Math.random() * keys.length)];
+}
+
+let drawnLetters = [];
```

Comment:
Because drawnLetters is only used in the drawLetters function, I would make the variable local to the drawLetters function. Also, we add values to it, but we never reassign the reference to drawnLetters, so const may be the better choice here! Always prefer const over let. If you accidentally try to reassign a variable that was declared with const, then the JS interpreter will let you know and you can change it to let.